src/design/@generated/themes/dark.constants.ts

```diff
@@ -80,3 +80,4 @@ export const keyboardTextColor = 'rgba(255, 255, 255, .8)';
 export const keyboardBackgroundColor = 'rgba(255, 255, 255, .05)';
 export const keyboardBorderColor = 'rgba(255, 255, 255, .1)';
 export const linkTextColor = '#00C2FF';
+export const goalBackgroundHover = 'rgba(255, 255, 255, .08)';
```

Comment:
Use one of this colors https://github.com/taskany-inc/issues/blob/main/src/design/palette.js#L15

Examples: https://github.com/taskany-inc/issues/blob/91d6769c978c409013546005f484ddac0388fdc1/src/components/Tag.tsx#L4